ref/general/observability-monitoring.adoc

```diff
@@ -0,0 +1,65 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: Observability helps you understand system behavior by processing and monitoring the data the system generates through metrics, logging, tracing, and health checks, to get the information you need to troubleshoot failure
+:page-categories: MicroProfile
+:seo-title: Observability helps you understand system behavior by processing and monitoring system data
+:seo-description: Microservice architecture is a popular approach for building cloud-native applications in which each capability is developed as an independent service. It enables small, autonomous teams to develop, deploy, and scale their respective services independently.
+= Observability and monitoring
+
+Observability helps you understand system behavior by processing and monitoring the data the system generates through metrics, logging, tracing, and health checks, to get the information you need to troubleshoot failure.
+
+In a microservices environment, your application needs to be functional as a whole, though individual service instances can start or stop to adjust for capacity requirements or to recover from failure.
+To monitor this fluid system each entity must provide appropriate data for problem detection.
+Metrics collect numerical information from the application and platform.
+Whereas, logs collect textual information.
+Tracing, on the other hand, captures the end-to-end flow of a request through the system.
+Health Checks help orchestrators perform automated actions to maintain overall system health.
+
+== Metrics
+
+Metrics are numerical measurements that are captured and processed to identify and respond to potential issues before they escalate and cause more serious problems.
+In distributed system metrics, three are factors are involved; producers, aggregators, and processors.
+
+Producers, for example, applications, can be directly involved in producing metrics. Also, other infrastructure might passively observe or actively instrument the application to produce metrics on its behalf.
+What happens next depends on the aggregator.
+
+In the case of aggregators, for example, Prometheus, metrics are transferred from the producer to the aggregator by way of either `push` or `pull` mechanisms.
+Some aggregators, expect the application to connect to the aggregator to transmit data.
+The connection information for the aggregator must be distributed to all of the application processes that are measured.
+Other aggregators periodically connect to a known endpoint to gather metrics data.
+This requires the producer to define and provide an endpoint that can be scraped, and for the aggregator to be told where the endpoints are.
```

Comment:
Change "This requires the producer" to something like "This action requires the producer" to avoid any possible confusion about what "This" is referring to.